ide-support/revdeploylibraryandroid.livecodescript

```diff
@@ -169,6 +169,11 @@ function deployIsValidJDK pPath
       return false
    end if
    
+   if not pPath contains "jdk1.8" then
```

Comment:
If they don't have new tools - do they need 1.8? i.e. Shoudl this check be done after we've determined where sdklib.jar is?